### DISCUSSION TITLE
Potential Problems with the Usage of Data and GIS

### DISCUSSION THREAD

**Discussion by okcirE** [albertkun/21S-ASIAAM-191A#39] [Past Thinking Caps]
Two problems of Data and GIS comes down to limitations in data collection and implicit bias that can occur. In my humble opinion, I don't agree with inferences made on a population that is concluded based on a small data sample. The causes of this method are limited budget and workforce to collect all data in the population. However, some conclusions may not portray the correct characteristics/behavior in the given population. The second problem is an implicit bias that can be perpetuated from either machine/program or method of data collection. This bias can skew results that are inaccurate and may leave certain populations behind in terms of progress. For example, the lower socioeconomic status population could receive less government aid because the data failed to include them. - Ericko Halim

**Comment by rh5140** [comment]
You bring up an important point about how results gets skewed by sampling error! I feel like something that should be included with any reference to a conclusion drawn from a data sample should be more information on how that data is collected. I know a lot of online articles would just link the statistic to the paper it's from, but that depends on if the reader takes the time to click the link and read through it. The context should be readily available within the article instead of only externally so people really understand what flaws there might be while reading.

**Comment by alizezila** [comment]
Thanks for sharing! You're absolutely right about the limitations of data collection and how that can affect what is presented in the final product. It is important that our sampled data is wide, diverse, and accurately displays the characteristics of the population we are interested in. Without fully realizing a proper data sample, we cannot accurately represent a population in our data our any visual representations of that data.
